nbs/00_utils.ipynb

```diff
@@ -12,17 +12,27 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    test_fail(set_config, kwargs={'rng_reserve_size': -1}, contains='must be non-negative')
</pre><p>Adjust test cases accordingly. Test cases before this line looks good.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/jax-relax/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>You should probably change your fourth test cases. I think we shouldn't change the behavior of <code>Config</code> if nothing is passed to <code>set_config</code>.</p><p> </p><p>So the test case should be</p><p> </p><p><code></code><code>python</p><p>assert get_config().rng_reserve_size == 2 and get_config().global_seed == 234</p><p></code><code></code></p>

Reply:
I have changed the implementation so that if a user passes no arguments to set config, then no change occurs and the previous values are still valid.